src/fah/webclient/index.html

```diff
@@ -3,481 +3,465 @@
-        <div id="box-points">
-          <span>Points earned</span>
-          <div class="clearfix">
-            <span id="box-points-counter">Unknown</span>
-            <a id="box-user-stats-link" href="#">(See stats)</a></span>
-          </div>
-          <div id="team-points">
-            <span id="box-points-counter-contributed">Unknown</span>
-            go to Team <span id="box-points-team">Unknown</span>'s total of
-            <span id="box-points-counter-team">Unknown</span>.
-          </div>
-          <div id="no-team">Consider joining a team.</div>
         </div>
-       </div>
-    </div>
-  </div>
-
-  <div class="row panel-pu" id="slots">
-      <ul class="box-pu"></ul>
-  </div>
-
-  <div class="row">
-    <div class="col-50">
-      <div class="col-left col-left-pu">
+      </div>
 
-        <div id="box-status" class="clearfix">
-          <div id="box-status-msg">Loading...</div>
-          <div id="box-status-points">
-            <span class="hlarge">-</span> Points per day
-          </div>
-        </div>
-        <p>My computer has <span id="box-status-time">unknown time</span> to
-        complete this work unit.</p>
+      <div class="row panel-pu" id="slots">
+          <ul class="box-pu"></ul>
+      </div>
 
-        <div id="box-all-slots">
-          <div id="box-power">
-            <div class="row">
-              <div class="col-50">
-                <div id="box-power-slider">
-                  <span class="hlarge">Power</span>
-                  <div id="slider"></div>
-                  <ul id="labels" class="clearfix">
-                    <li class="label-1">Light</li>
-                    <li class="label-2">Medium</li>
-                    <li class="label-3">Full</li>
-                  </ul>
-                </div>
+      <div class="row">
+        <div class="one-half column">
+          <div class="col-left col-left-pu">
 
+            <div id="box-status" class="clearfix">
+              <div id="box-status-msg">Loading...</div>
+              <div id="box-status-points">
+                <span class="hlarge">-</span> Points per day
               </div>
-              <div class="col-50">
-                <div id="box-idle-selector">
-                  <span class="hlarge">When</span>
-                  <ul class="radio-selector">
-                    <li>
-                      <input id="not_idle" name="idle" type="radio" value="1"
-                    	     checked="checked">
-                      <label for="not_idle">
-                        <span></span>While I'm working
-                      </label>
-                    </li>
-                    <li>
-                      <input id="idle" name="idle" type="radio" value="2">
-                      <label for="idle"><span></span>Only when idle</label>
-                    </li>
-                  </ul>
+            </div>
+            <p>My computer has <span id="box-status-time unknown">unknown time</span> to
```

Comment:
As I see `unknown` class wasn't hardcoded before. What is the reason for adding it now?

Reply:
Oh, I see.
You remove it in JS code 😃
https://github.com/FoldingAtHome/fah-web-client/pull/14/files?diff=split&w=1#diff-57086a2aa886379d4ee4c73304909162R477

Reply:
Is it normal to have this on an `id` and not on a `class` ?